src/wavesurfer.ts

```diff
@@ -266,6 +266,15 @@ class WaveSurfer extends Player<WaveSurferEvents> {
           this.emit('click', relativeX, relativeY)
         }
       }),
+       // Right click
+    this.renderer.on('contextmenu', (relativeX, relativeY, event: MouseEvent) => {
+      event.preventDefault();  // Prevent default context menu
+      if (this.options.interact && event.button === 2) {  // Right-click
+        this.seekTo(relativeX)
+        this.emit('interaction', relativeX * this.getDuration())
+        this.emit('click', relativeX, relativeY)  // Trigger same event as left-click
```

Comment:
Why don't we emit a diff event? something like `rightclick` and listen to it and allow user of package to trigger whatever callback they want on that event?

Reply:
we can do that ig. but the current situation doesn't need it. lmk if i should try doing that